modules/client-configuration/pages/client-upgrades-uyuni.adoc

```diff
@@ -10,11 +10,11 @@ In this section, we use openSUSE Leap as an example.
 +
 ----
 spacewalk-common-channels \
-opensuse_leap15_4 \
```

Comment:
Let's not change the default example yet. Leap 15.5 is still Alpha

Reply:
Undone